tests/toolchain/install.pm

```diff
@@ -0,0 +1,46 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    send_key 'ctrl-alt-f4';
+    assert_screen 'tty4-selected';
+    assert_screen 'text-login';
+    type_string "root\n";
+    assert_screen 'password-prompt', 10;
+    type_password;
+    send_key "ret";
+    assert_screen 'text-logged-in';
+    # disable packagekitd
+    type_string "chmod 444 /usr/lib/packagekitd\n"; # packagekitd will be not executable
+    type_string "killall packagekitd\n";
```

Comment:
I know you chmod 444 approach worked for SLE11 but maybe this sle-12 targeted test should use the systemd approach like lines 27-28 on https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/consoletest_setup.pm